src/pages/Dashboard/PieChart.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useRef, useState } from 'react';
+import ApexCharts from 'apexcharts';
+import { Box } from '@chakra-ui/react';
+
+interface User {
+  id: number;
+  name: string;
+}
+
+const ChartComponent = () => {
+  const [data, setData] = useState<number[]>([]);
+  const [labels, setLabels] = useState<string[]>([]);
+  const chartRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    // Função para buscar dados do backend
+    const fetchData = async () => {
+      try {
+        const response = await fetch('http://localhost:8000/api/users');
```

Comment:
E aqui coloque aquela variavel host
```suggestion
        const response = await fetch(`${host}/users`);
```